src/Tvheadend.cpp

```diff
@@ -665,14 +706,20 @@ PVR_ERROR CTvheadend::GetTimerTypes ( PVR_TIMER_TYPE types[], int *size )
 
   /* PVR_Timer.iPreventDuplicateEpisodes values and presentation.*/
   static std::vector< std::pair<int, std::string> > deDupValues;
-  if (deDupValues.size() == 0)
```

Comment:
This will not work as we do a version check a few lines below, and htspversion = 0 the first time. 
It's working for the other methods, so I guess Kodi will reject duplicates.